src/account/actions.ts

```diff
@@ -30,6 +30,7 @@ export enum Actions {
   DISMISS_KEEP_SUPERCHARGING = 'ACCOUNT/DISMISS_KEEP_SUPERCHARGING',
   DISMISS_START_SUPERCHARGING = 'ACCOUNT/DISMISS_START_SUPERCHARGING',
   SAVE_SIGNED_MESSAGE = 'ACCOUNT/SAVE_SIGNED_MESSAGE',
+  SET_GOLD_EDUCATION_COMPLETED = 'ACCOUNT/SET_GOLD_EDUCATION_COMPLETED',
```

Comment:
nit: Gold term is really old :D
```suggestion
  SET_CELO_EDUCATION_COMPLETED = 'ACCOUNT/SET_CELO_EDUCATION_COMPLETED',
```